src/components/CountryPerformanceOnRisk.js

```diff
@@ -55,11 +55,11 @@ export class CountryPerformanceOnRisk extends Component {
 
   async componentDidMount() {
     await this.props.dispatch(setViews(this.props.serverProps))
-    this.props.dispatch(fetchData(
+    this.props.dispatch(fetchDataIfNeeded(
       this.props.views[1].country,
       this.props.views[1].risk
```

Comment:
We still have views hard coded. We should iterate over `this.props.views` and fetch data of in that way. Smth like:
```
await this.props.dispatch(setViews(this.props.serverProps))
Object.values(this.props.views).forEach(view => {
  this.props.dispatch(fetchDataIfNeeded(view.country, view.risk))
})
```